package/thingino-webui/files/var/www/x/config-wireguard.cgi

```diff
@@ -58,7 +75,12 @@ fi
 	</div>
 	<div class="col">
 		<h3>Environment Settings</h3>
-		<% ex "fw_printenv | grep -E '^wg_' | sed -Ee 's/(key|psk)=.*$/\1=[__redacted__]/' | sort" %>
```

Comment:
please keep `sort` in place. it is much easier to skim the list when it is sorted alphabetically.

Reply:
done